deploy/group_vars/all.yml

```diff
@@ -96,7 +96,7 @@ k8s_iam_users: [copelco]
 # https://github.com/kubernetes/ingress-nginx/releases
 k8s_ingress_nginx_chart_version: "4.4.2"
```

Comment:
Is there a reason to leave this at 4.4.2?

Reply:
I remember Colin and I talking about it, but don't remember the specific reason.